src/SUMMARY.md

```diff
@@ -142,6 +142,7 @@ Space Station 14
 	- [Player Interaction](en/space-station-14/areas/core/player-interaction.md)
 		- [PR Guidelines](en/space-station-14/areas/core/player-interaction/guidelines.md)
 			- [PDA Messaging](en/space-station-14/areas/core/player-interaction/proposals/pda-messaging.md)
+  	- [PAI Expansion Slots](en/space-station-14/areas/core/player-interaction/proposals/pai-expansion-slots.md)
```

Comment:
uhh isnt the spacing sitll wrong? it should be under player interaction not at the same level

Reply:
idk it looks bad i think theres tabs mixed or phone github evil

Reply:
yeah try to fix it when you come to your pc or something

Reply:
its all good now i think